R/getPrevalence.R

```diff
@@ -594,23 +585,22 @@ setMethod("getPrevalentAbundance", signature = c(x = "SummarizedExperiment"),
 
 ############################# agglomerateByPrevalence ##########################
 
-#' @rdname getPrevalence
-#' @aliases mergeFeaturesByPrevalence
+#' @rdname agglomerate-methods
+#' 
+#' @details
+#' \code{agglomerateByPrevalence} sums up the values of assays at the taxonomic 
+#' level specified by \code{rank} (by default the highest taxonomic level 
+#' available) and selects the summed results that exceed the given population 
+#' prevalence at the given detection level. The other summed values (below the 
+#' threshold) are agglomerated in an additional row taking the name indicated by
+#' \code{other_label} (by default "Other").
+#' 
```

Comment:
Is it possible to add examples here instead of agglomerate.R? I am not sure if it works like that. Are the examples then included in agglomerate.methods.Rd along with examples from agglomerate.R?

Reply:
I moved it here and it looks like it works well.